devops/pypi-release-new.yml

```diff
@@ -46,7 +46,7 @@ stages:
     parameters:
       platforms:  { Linux: ubuntu-latest, Windows: windows-latest }
       installationType: 'PipLocal'
-      pyVersions: [3.7, 3.8, 3.9]
+      pyVersions: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
Nitpick.... quote all of these? And/or perhaps add a comment about `# 3.10 not 3.1` ?